### PR TITLE
fix: modified i18n and dependencies

### PR DESCRIPTION
**Pull request by qinluhe**

### CODE REVIEW

File: src/components/ui/textarea.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import * as React from "react"
+
+import { cn } from "@/lib/utils"
+
+const Textarea = React.forwardRef<
+  HTMLTextAreaElement,
+  React.ComponentProps<"textarea">
+>(({ className, ...props }, ref) => {
+  return (
+    <textarea
```
Comment (nik-lylo) [inline src/components/ui/textarea.tsx:10]: The Textarea component does not look the same as we have in the design. The Textarea component should be auto-resizable. Manual vertical resizing should be disabled.

File: src/components/ui/textarea.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+      className={cn(
+        "flex min-h-[60px] w-full rounded-md border border-input bg-transparent px-3 py-2 text-foreground shadow-sm placeholder:text-muted-foreground focus-visible:outline-none focus-visible:ring-1 focus-visible:ring-ring disabled:cursor-not-allowed disabled:opacity-50 md:text-sm",
```
Comment (nik-lylo) [inline src/components/ui/textarea.tsx:12]: placeholder's color does not match the design

File: src/chat/components/button-icon/index.tsx [hunk 3]
```diff
@@ -15,41 +16,25 @@ interface Props extends ComponentProps<'button'> {
...
-          'hover:bg-ch-fill-hover disabled:text-ch-text-disabled flex h-6 items-center gap-[1px] rounded-lg p-1 transition-colors disabled:bg-transparent',
-          { [activeClass]: active },
-        ])}
-        type='button'
+    <Tooltip>
+      <TooltipTrigger asChild>
+      <Button
+        variant={'ghost'}
+        size={'icon'}
+        color={'default'}
```
Comment (nik-lylo) [inline src/chat/components/button-icon/index.tsx:29]: Icon's color does not match the design 
image

File: src/chat/components/button-bar-ai-popover/index.tsx [hunk 4]
```diff
@@ -83,86 +79,72 @@ const ButtonBarAIPopover: FC<Props> = ({
...
-                onClick={() => handleOptionChange(option.value)}
-              >
-                <div>{option.name}</div>
-                {activeOption === option.value && (
-                  <IconCheck className='text-ch-icon-blue h-4 w-4 flex-shrink-0' />
-                )}
-              </div>
-            ))}
+      </PopoverTrigger>
+      <PopoverContent
```
Comment (nik-lylo) [inline src/chat/components/button-bar-ai-popover/index.tsx:99]: popover's border and box shadow do not match the design

image

File: src/chat/components/button-icon/index.tsx [hunk 5]
```diff
@@ -15,41 +16,25 @@ interface Props extends ComponentProps<'button'> {
...
+        startIcon={icon}
         {...rest}
       >
-        <div
-          className={clsx(
-            iconMainWrapClass
```
Comment (nik-lylo) [inline src/chat/components/button-icon/index.tsx:43]: If we do not use the iconMainWrapClass prop, consider removing this prop as it caused the error.
image

**Comment by qinluhe** [comment]
I will modify the color values ​​uniformly, and there are still many components to replace. I will replace them uniformly later.

**Pull request closed without merging**